format/File.fbs

```diff
@@ -37,10 +37,7 @@ struct Block {
 
   offset: long;
 
-  metaDataLength: int;
-
-  bodyLength: long;
-
+  length: int;
```

Comment:
The problem with combining the metadata and body length is that it makes it difficult / impossible to do partial reads / field projections in a record batch. If you can inspect the metadata without reading the body, you can determine the byte ranges you need to read only a certain subset of fields.